aucoalesce/testdata/random-internet.json.golden

```diff
@@ -50,6 +50,16 @@
           "type": [
             "end"
           ]
+        },
+        "user": {
```

Comment:
Are these the result of the encoder not honoring `omitempty` for zero value structs?

Reply:
Yes. I couldn't find a workaround